setpass/api.py

```diff
@@ -166,5 +168,46 @@ def add(user_id):
     return Response(response=user.token, status=200)
 
 
+@wsgi.app.route('/reset', methods=['GET'])
+def view_reset_form():
+    return render_template('reset_form.html')
+
```

Comment:
I've thought about it a little -- probably better to keep them separate. 

Originally I thought since they have the same `route` we should combine them for brevity ("don't repeat yourself") or clarity (debatable). But if they are separate, having distinct function names `view_reset_form` and `reset_password` allows the code to be self-documenting.